src/resolvers/git.cr

```diff
@@ -305,7 +305,7 @@ module Shards
     end
 
     private def mirror_repository
-      run_in_current_folder "git clone --mirror --quiet -- #{Helpers::Path.escape(git_url)} #{local_path}"
+      run_in_current_folder "GIT_ASKPASS=${GIT_ASKPASS:-/usr/bin/test} git clone --mirror --quiet -- #{Helpers::Path.escape(git_url)} #{local_path}"
```

Comment:
In OSX 

```
$ which test
/bin/test
```

Reply:
Does it  need to be an absolute  path or does it do PATH lookup?  Maybe  `${GIT_ASKPASS:-false}` would be most portable.